e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/B_PublishMultipleNodesOrchestratedTestTheory.cs

```diff
@@ -0,0 +1,387 @@
+            Assert.NotNull(response);
+
+            if (!response.IsSuccessful) {
+                _output.WriteLine($"StatusCode: {response.StatusCode}");
+                _output.WriteLine($"ErrorMessage: {response.ErrorMessage}");
+                Assert.True(response.IsSuccessful, "POST /publisher/v2/publish/{endpointId}/bulk failed!");
+            }
+        }
+
+        [Fact, PriorityOrder(53)]
+        public void Test_GetListOfJobs_Expect_JobWithEndpointId() {
+
+            // used if running test cases separately (during development)
+            if (string.IsNullOrWhiteSpace(_context.OpcUaEndpointId)) {
+                Test_PrepareTestDeploymentForTestCase_Expect_Success();
+                Assert.False(string.IsNullOrWhiteSpace(_context.OpcUaEndpointId));
+            }
+
+            var cts = new CancellationTokenSource(TestConstants.MaxTestTimeoutMilliseconds);
+            var accessToken = TestHelper.GetTokenAsync(_context, cts.Token).GetAwaiter().GetResult();
+            var client = new RestClient(_context.IIoTPlatformConfigHubConfig.BaseUrl) {
+                Timeout = TestConstants.DefaultTimeoutInMilliseconds
+            };
+
+            var request = new RestRequest(Method.GET);
+            request.AddHeader(TestConstants.HttpHeaderNames.Authorization, accessToken);
+            request.Resource = TestConstants.APIRoutes.PublisherJobs;
+
+            var response = client.ExecuteAsync(request, cts.Token).GetAwaiter().GetResult();
+            Assert.NotNull(response);
+
+            if (!response.IsSuccessful) {
+                _output.WriteLine($"StatusCode: {response.StatusCode}");
+                _output.WriteLine($"ErrorMessage: {response.ErrorMessage}");
+                Assert.True(response.IsSuccessful, "GET /publisher/v2/jobs failed!");
+            }
+
+            dynamic json = JsonConvert.DeserializeObject<ExpandoObject>(response.Content, new ExpandoObjectConverter());
+
+            bool found = false;
+            for (int jobIndex = 0; jobIndex < (int)json.jobs.Count; jobIndex++) {
+                var id = (string)json.jobs[jobIndex].id;
+                if (id == _context.OpcUaEndpointId) {
+                    found = true;
+                    break;
+                }
+            }
+            Assert.True(found, "Publishing Job was not created!");
+        }
+
+        [Fact, PriorityOrder(54)]
+        public void Test_VerifyDataAvailableAtIoTHub() {
+
+            var cts = new CancellationTokenSource(TestConstants.MaxTestTimeoutMilliseconds);
+
+            //use test event processor to verify data send to IoT Hub (expected* set to zero as data gap analysis is not part of this test case)
+            TestHelper.StartMonitoringIncomingMessagesAsync(_context, 50, 1000, 90_000_000, cts.Token).GetAwaiter().GetResult();
+            // wait some time to generate events to process
+            Task.Delay(180 * 1000, cts.Token).GetAwaiter().GetResult();
```

Comment:
>180 * 1000 [](start = 23, length = 10)

magic number

Reply:
true; added comment why the magic number is used